src/main/java/fi/metatavu/kuntaapi/server/index/IndexableContact.java

```diff
@@ -0,0 +1,160 @@
+package fi.metatavu.kuntaapi.server.index;
+
+import java.util.List;
+
+public class IndexableContact implements Indexable {
+
+  @Field(index = "not_analyzed", store = true, type = "long")
+  private Long orderIndex;
+
+  @Field(index = "not_analyzed", store = true)
+  private String contactId;
+  
+  @Field (index = "not_analyzed", store = true)
+  private String organizationId;
+
+  private String displayName;
+
+  @Field (index = "not_analyzed", store = true)
+  private String displayNameUT;
+
+  private String firstName;
+
+  private String lastName;
+
+  private String title;
+
+  private String organization;
+
+  private List<String> organizationUnits;
+
+  private List<String> additionalInformations;
+
+  private List<String> emails;
+
+  private List<String> phoneNumbers;
+  
+  @Field (index = "not_analyzed", store = true)
+  private Boolean privateContact;
+
+  @Override
+  public String getId() {
+    return contactId;
+  }
+
+  @Override
+  public String getType() {
+    return "contact";
+  }
+
+  @Override
+  public Long getOrderIndex() {
+    return orderIndex;
+  }
+
+  public void setOrderIndex(Long orderIndex) {
+    this.orderIndex = orderIndex;
+  }
+
+  public String getContactId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 41. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)